autoload/dirvish.vim

```diff
@@ -83,6 +83,7 @@ function! dirvish#shdo(l1, l2, cmd)
   write
   if executable('chmod')
     call system('chmod u+x '.tmpfile)
+    edit
```

Comment:
autocmds already were triggered in the `execute 'split' tmpfile '` case above. The eunuch case is justified because the file is intentionally be re-loaded in a different state... I guess the same could be said for this case.

> (I have seen some issue with regard to vim-diminactive not undimming when using dirvish - probably due to some usage of noautocmd

I do want to remove most of the usages of noautocmd in dirvish. I will take another look at it.